extensions/ql-vscode/src/local-queries/query-runner.ts

```diff
@@ -0,0 +1,68 @@
+// TODO: maybe another filename?
```

Comment:
I think calling it `run-query.ts` would be perfectly fine and would probably be a more easily searchable name.